README.md

```diff
@@ -19,9 +19,9 @@ Python 2.7 and 3.4+
 If the python package is hosted on a repository, you can install directly using:
 
 ```sh
-pip install git+https://github.com/orlangure/gnomock-python-sdk.git
+pip install git+https://github.com/GIT_USER_ID/GIT_REPO_ID.git
```

Comment:
To avoid this issue, you should specify two more options when generating code:

```
openapi-generator generate -i swagger.yaml -g python -o ../../gnomock-python-sdk \
    -c config/python.yaml --git-user-id orlangure --git-repo-id gnomock-python-sdk
```